vast/integration/reference/server-zeek-conn-log/step_07.ref

```diff
@@ -8,8 +8,6 @@
 {"path":["importer","ids","available"],"type":"number"}
 {"path":["importer","ids","block","end"],"type":"number"}
 {"path":["importer","ids","block","next"],"type":"number"}
-{"path":["index","backlog","num-low-priority"],"type":"number"}
-{"path":["index","backlog","num-normal-priority"],"type":"number"}
```

Comment:
This removal needs a separate changelog entry.

Reply:
We decided to bring this back in the status output and the metrics.